### PR TITLE
Hw2

### PR DESCRIPTION
**Pull request by Yevhenii-Bezeha**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,104 +1,46 @@
-# Logs
```
Comment (MMaksymivUa) [inline .gitignore:1]: Too many changes. What is the purpose of updating gitignore file ???

File: src/app/components/footer/footer.component.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
+.footer {
+  background: #1a1e25;
+  color: #868c96;
+  width: 24px;
+  position: absolute;
```
Comment (MMaksymivUa) [inline src/app/components/footer/footer.component.scss:5]: fixed

File: src/app/components/form/form.component.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
+<form class="form" [formGroup]="postForm" (ngSubmit)="onSubmit()">
```
Comment (MMaksymivUa) [inline src/app/components/form/form.component.html:1]: good job, but please, use better naming, for example, article-form.component

File: src/app/components/main/list/list-item/list-item.component.scss [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  &__btn {
+    background-color: inherit;
+    cursor: pointer;
+    padding: 2px 3px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 19px;
+    border-radius: 6px;
+  }
```
Comment (MMaksymivUa) [inline src/app/components/main/list/list-item/list-item.component.scss:18]: create mixin button or add to general css/scss/less file

File: src/app/components/material/material.module.ts [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  MatSidenavModule,
+  MatToolbarModule,
+  MatIconModule,
+  MatListModule,
+  MatCardModule,
+  MatInputModule,
+];
+
+@NgModule({
+  imports: [materialComponents],
```
Comment (MMaksymivUa) [inline src/app/components/material/material.module.ts:21]: use or imports: materialComponents or imports: [...materialComponents]

File: src/app/components/modal/modal.component.scss [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+  width: 100%;
+  background-color: rgba($color: #000000, $alpha: 0.3);
+  z-index: 1;
+  overflow: hidden;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.Modal {
```
Comment (MMaksymivUa) [inline src/app/components/modal/modal.component.scss:14]: bad name of class

File: src/app/components/modal/modal.component.scss [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+  position: absolute;
+  border-radius: 14px;
+  z-index: 999;
```
Comment (MMaksymivUa) [inline src/app/components/modal/modal.component.scss:17]: please create variable for z-index

File: src/app/components/sidebar/sidebar-nav/sidebar-nav.component.scss [hunk 8]
```diff
@@ -0,0 +1,15 @@
+.nav {
+  margin-bottom: 30px;
+
+  &__text {
+    margin: 0;
+    font-size: 14px;
+    font-weight: 700;
+    color: rgba($color: #000000, $alpha: 0.3);
```
Comment (MMaksymivUa) [inline src/app/components/sidebar/sidebar-nav/sidebar-nav.component.scss:8]: it is a bad approach, convert it to hex color and use it directly

File: src/app/services/postsService.ts [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+      authorName: 'John Dow',
+      date: 'November 2, 2021',
+      title: 'Natural language interface accessibility',
+      description: 'Spoken interaction with mobile devices and consumer',
+      likes: 0,
+    },
+  ];
+
+  getPosts() {
+    return this.posts.slice();
```
Comment (MMaksymivUa) [inline src/app/services/postsService.ts:27]: ??? slice?

File: .gitignore (hunk 1 above)
Comment (Yevhenii-Bezeha) [reply .gitignore:1]: Plugin in Webstorm (.ignore) did it . I installed it during this PL.

File: src/app/components/footer/footer.component.scss (hunk 2 above)
Comment (Yevhenii-Bezeha) [reply src/app/components/footer/footer.component.scss:5]: Fixed

File: src/app/components/form/form.component.html (hunk 3 above)
Comment (Yevhenii-Bezeha) [reply src/app/components/form/form.component.html:1]: Good. Thanks. Fixed

File: src/app/components/main/list/list-item/list-item.component.scss (hunk 4 above)
Comment (Yevhenii-Bezeha) [reply src/app/components/main/list/list-item/list-item.component.scss:18]: Done

File: src/app/components/material/material.module.ts (hunk 5 above)
Comment (Yevhenii-Bezeha) [reply src/app/components/material/material.module.ts:21]: Good. Done

File: src/app/components/modal/modal.component.scss (hunk 6 above)
Comment (Yevhenii-Bezeha) [reply src/app/components/modal/modal.component.scss:14]: Changed, thanks for the review

File: src/app/components/modal/modal.component.scss (hunk 7 above)
Comment (Yevhenii-Bezeha) [reply src/app/components/modal/modal.component.scss:17]: Done

File: src/app/components/sidebar/sidebar-nav/sidebar-nav.component.scss (hunk 8 above)
Comment (Yevhenii-Bezeha) [reply src/app/components/sidebar/sidebar-nav/sidebar-nav.component.scss:8]: Fixed

File: src/app/services/postsService.ts (hunk 9 above)
Comment (Yevhenii-Bezeha) [reply src/app/services/postsService.ts:27]: It was made by mistake, changed it using SPREAD

**Pull request merged**